usr/share/lib/ipa/tests/SLES/EC2/test_sles_ec2_billing_code.py

```diff
@@ -0,0 +1,4 @@
+
+
+def test_sles_ec2_billing_code(get_ec2_billing_products):
```

Comment:
Please add the following.

"""This is really ONLY for SLES. All other on-demand products are sold through the Marketplace and they do not have a billing-product-code."""